to ensure that correct compiler commands are used
Kenneth Hoste (HPC-UGent), updated for v3.6.2 by Jasper Grimm (UoY)
diff -ruN gdal-3.6.2.orig/swig/python/setup.py.in gdal-3.6.2/swig/python/setup.py.in
--- gdal-3.6.2.orig/swig/python/setup.py.in	2023-01-24 10:50:11.000000000 +0000
+++ gdal-3.6.2/swig/python/setup.py.in	2023-01-02 14:38:17.000000000 +0000
@@ -16,6 +16,12 @@
 from setuptools import find_packages
 from setuptools import Extension
 
+for key in ['CC', 'CXX']:
+    seqkey = '%s_SEQ' % key
+    if seqkey in os.environ:
+        os.environ[key] = os.environ[seqkey]
+        print('$%s set to %s (via $%s)' % (key, os.environ[key], seqkey))
+
 # If CXX is defined in the environment, it will be used to link the .so
 # but setuptools will be confused if it is made of several words like 'ccache g++'
 # and it will try to use only the first word.
